### ISSUE TITLE
Can paragraph be used for indel from 2 bp to 30 bp? 

### ISSUE THREAD

**Issue by 349539427** [Illumina/paragraph#62]
Can paragraph be used for indel from 2 bp to 30 bp?

**Comment by traxexx** [comment]
Yes. The graph model should work for any length of variants. We tested it out before and it works fine for most indels.
But some optimizations might not work for small indels, which means the performance could be lower than good SVs.
